sidestream-tech/unified-auctions-ui
# Chaoslabs testing setup

Closes #83 

### Testing scenario

1. Clone the repo `git@github.com:sidestream-tech/unified-auctions-ui.git`
2. Cd into the frontend `cd unified-auctions-ui/frontend`
3. Create `.env` file with `CHAOSLABS_ACCESS_TOKEN=` and `CHAOSLABS_SIMULATION_IDS=` values
    Or use `secret export --format=dotenv auction-ui/localhost/frontend -o=.env` if you have access to our secrets
4. Install dependencies `npm run ci`
5. Start frontend in the dev mode `npm run dev`
6. Open http://localhost:3000/collateral?network=custom
7. Click one of the simulation buttons (see example below)
8. Wait for the simulation to load
8. If the simulation started successfully, the input should be prefilled with the url, click `Set RPC endpoint`

### Screenshot
<img width="542" alt="Screenshot 2022-02-07 at 18 41 28" src="https://user-images.githubusercontent.com/3393626/152842528-242749a9-ff7b-429f-80c8-1b52895a8f48.png">
